app/printTemplates/summary/local/ca_objects_summary.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/* ----------------------------------------------------------------------
+ * app/templates/summary/local/ca_objects_summary.php
```

Comment:
This is a copy of https://github.com/kehh/providence/blob/c7116f5a337eff3167d3340b6717571fd4ea8266/app/printTemplates/summary/ca_objects_summary.php with some modifications